android/app/src/main/java/com/expensify/chat/MainApplication.java

```diff
@@ -18,10 +18,12 @@
 import com.google.firebase.crashlytics.FirebaseCrashlytics;
 
 import java.lang.reflect.Field;
-import java.lang.reflect.InvocationTargetException;
 import java.util.List;
 
 public class MainApplication extends MultiDexApplication implements ReactApplication {
+
+  private static Context context = null;
```

Comment:
Holding context refs is dangerous and can lead to memory leaks. Will suggest other solutions below.

Reply:
Hmm so I added this static context ref because the Airship's `onNotificationDismissed` listener doesn't provide one. 

They do provide it when we construct the listener and provider [here](https://github.com/Expensify/App/blob/main/android/app/src/main/java/com/expensify/chat/customairshipextender/CustomAirshipExtender.java#L12) so we could save that in a non-static field in the listener. How does that sound?

Reply:
That's better, but it would be ideal to not hold references at all if possible in long-living classes, as it's an easy way to introduce memory leaks. 

> because the Airship's onNotificationDismissed listener doesn't provide one.

I can't find where exactly within `onNotificationDismissed` we need context? Let me know what we need it for and there might be alternative